tools/validators/instance_validator/tests/instance_parser_test.py

```diff
@@ -224,7 +224,7 @@ def testEntityBlock_ValidUpdateMaskValue_Success(self):
         'SDC_EXT-17-GUID': {
             'type': 'HVAC/SDC_EXT',
             'code': 'SDC_EXT-17',
-            'cloud_device_id': 'foobar',
+            'cloud_device_id': '2619178366980754',
```

Comment:
change cloud_device_id to something more generic like 1234567890123456

Reply:
Done. My lord, I am done.